pom.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.mule.extensions</groupId>
+        <artifactId>mule-core-modules-parent</artifactId>
+        <version>1.1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.mule.modules</groupId>
+    <artifactId>mule-xml-module</artifactId>
+    <packaging>mule-extension</packaging>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <name>XML Module</name>
+    <description>
+        A Mule Module which provides functionality to handle XML. This module should be used for advanced use cases or use cases
+        associated with XML standards such as XSLT, XPath and XQuery.
+        Take in mind that DataWeave is not only perfectly capable to handle most of those use cases and can replace those standards the majority
+        of the times, but it's also our recommended approach.
+    </description>
+
+    <properties>
+        <formatterConfigPath>formatter.xml</formatterConfigPath>
+    </properties>
+
+    <dependencies>
+        <!-- XSLT and Xpath Support -->
+        <dependency>
+            <groupId>net.sf.saxon</groupId>
+            <artifactId>Saxon-HE</artifactId>
+            <version>9.8.0-6</version>
```

Comment:
what for? they're not reused

Reply:
- convention 
- is easier to read and update when required

Reply:
none are true. We don't have this convention and this doesn't make it easier to update. The value of defining a property is when the same version is referenced multiple times. Check our poms and that's the real convention that we follow. We do not consistently declare all versions as properties just because.